nsv13/code/modules/power/rbmk.dm

```diff
@@ -0,0 +1,840 @@
+		return TRUE
+	if(istype(W, /obj/item/sealant))
+		if(power >= 20)
+			to_chat(user, "<span class='notice'>You cannot repair [src] while it is running at above 20% power.</span>")
+			return FALSE
+		if(vessel_integrity >= 350)
+			to_chat(user, "<span class='notice'>[src]'s seals are already in-tact, repairing them further would require a new set of seals.</span>")
+			return FALSE
+		if(vessel_integrity <= 0.5 * initial(vessel_integrity)) //Heavily damaged.
+			to_chat(user, "<span class='notice'>[src]'s reactor vessel is cracked and worn, you need to repair the cracks with a welder before you can repair the seals.</span>")
+			return FALSE
+		if(do_after(user, 5 SECONDS, target=src))
+			playsound(src, 'sound/effects/spray2.ogg', 50, 1, -6)
+			user.visible_message("<span class='warning'>[user] applies sealant to some of [src]'s worn out seals.</span>", "<span class='notice'>You apply sealant to some of [src]'s worn out seals.</span>")
+			vessel_integrity += 10
+			vessel_integrity = CLAMP(vessel_integrity, 0, initial(vessel_integrity))
+		return TRUE
+
+/obj/machinery/atmospherics/components/trinary/nuclear_reactor/welder_act(mob/living/user, obj/item/I)
+	if(power >= 20)
+		to_chat(user, "<span class='notice'>You can't repair [src] while it is running at above 20% power.</span>")
+		return FALSE
+	if(vessel_integrity > 0.5 * initial(vessel_integrity))
+		to_chat(user, "<span class='notice'>[src] is free from cracks. Further repairs must be carried out with flexi-seal sealant.</span>")
+		return FALSE
+	if(I.use_tool(src, user, 0, volume=40))
+		vessel_integrity += 20
+		to_chat(user, "<span class='notice'>You weld together some of [src]'s cracks. This'll do for now.</span>")
+	return TRUE
+
+//Admin procs to mess with the reaction environment.
+
+/obj/machinery/atmospherics/components/trinary/nuclear_reactor/proc/lazy_startup()
+	for(var/I=0;I<5;I++){
+		fuel_rods += new /obj/item/fuel_rod(src)
+	}
+	start_up()
+
+/obj/machinery/atmospherics/components/trinary/nuclear_reactor/proc/deplete()
+	for(var/obj/item/fuel_rod/FR in fuel_rods){
+		FR.depletion = 100
+	}
+
+/obj/machinery/atmospherics/components/trinary/nuclear_reactor/Initialize()
+	. = ..()
+	gas_absorption_effectiveness = rand(5, 6)/10 //All reactors are slightly different. This will result in you having to figure out what the balance is for K.
+	STOP_PROCESSING(SSmachines, src) //We'll handle this one ourselves.
+
+/obj/machinery/atmospherics/components/trinary/nuclear_reactor/Crossed(atom/movable/AM, oldloc)
+	. = ..()
+	if(isliving(AM) && temperature > 0)
+		var/mob/living/L = AM
+		L.adjust_bodytemperature(CLAMP(temperature, BODYTEMP_COOLING_MAX, BODYTEMP_HEATING_MAX)) //If you're on fire, you heat up!
+
+/obj/machinery/atmospherics/components/trinary/nuclear_reactor/process()
+	update_parents() //Update the pipenet to register new gas mixes
+	if(next_slowprocess < world.time)
```

Comment:
What the hell is this?

Reply:
I don't want it processing at atmos speed AKA multiple times per second.